js/components/controlBar.js

```diff
@@ -208,6 +208,18 @@ var ControlBar = React.createClass({
     });
   },
 
+  // WIKIA CHANGE
+  formatSecondsWithoutLeadingZero: function (timeInSeconds) {
```

Comment:
IMO this name is a little bit misleading, I thought that the result of this function will be seconds without leading '0' while it is actually the opposite.